tests/functional/testTelemetryView.js

```diff
@@ -35,7 +36,9 @@ describe('Telemetry view', function() {
     assert(await vpn.getSetting('glean-enabled') === 'true');
     await _getToTelemetryPage();
     await vpn.clickOnElement(telemetryScreen.DECLINE_TELEMETRY);
-    await vpn.wait();
+    await vpn.waitForCondition(
+        async () => await vpn.getSetting('telemetry-policy-shown') === 'true');
```

Comment:
love this use of getSetting checks here, will prove to be quicker and more stable